tutorials/2d/2d_sprite_animation.rst

```diff
@@ -0,0 +1,155 @@
+.. _doc_2d_sprite_animation:
+
+2D Sprite animation
```

Comment:
AFAIK this tutorial is compatible with 3.1 (i.e. does not cover the new Atlas packaging feature in 3.2). CC @cbscribe to confirm.